sigal/gallery.py

```diff
@@ -440,6 +443,13 @@ def thumbnail(self):
         return None
 
     @property
+    def rand_thumbnail(self):
+        try :
+            return join(self.name, random.choice(self.medias).thumbnail)
+        except IndexError:
+            return self.thumbnail
```

Comment:
Yes it is, I use in my template and it works fine. I was just pointing that the call to `url_from_path` is unnecessary because this function calls `thumbnail` attribute and thus return an url path (as `url_from_path` is called in `thubnail`).